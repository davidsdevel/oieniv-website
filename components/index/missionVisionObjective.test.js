/*eslint-env jest*/
import React from "react";
import MissionVisionObjective from "./missionVisionObjective";
import {shallow} from "enzyme";
import renderer from "react-test-renderer";

describe("MissionVisionObjective with Enzyme", () => {

	it("render three MissionVisionObjectiveChild", () => {
		const wrapper = shallow(<MissionVisionObjective/>);

		expect(wrapper.find("MissionVisionObjectiveChild")).toHaveLength(3);
	});
});

describe("MissionVisionObjective with Snapshot", () => {
	it("render without crash", () => {
		const component = renderer.create(<MissionVisionObjective/>);
		const tree = component.toJSON();

		expect(tree).toMatchSnapshot();
	});
});
