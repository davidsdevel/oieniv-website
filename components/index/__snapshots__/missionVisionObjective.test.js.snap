// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render MissionVisionObjective without crash 1`] = `
<div
  id="mission-main"
>
  <div>
    <div
      className="mission-container"
    >
      <img
        alt="OIENIV Misi贸n"
        src="/static/images/flag1.svg"
      />
      <h2>
        Misi贸n
      </h2>
      <p>
        Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod
		tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam,
		quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo
		consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse
		cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non
		proident, sunt in culpa qui officia deserunt mollit anim id est laborum.
      </p>
    </div>
    <div
      className="mission-container"
    >
      <img
        alt="OIENIV Visi贸n"
        src="/static/images/rocket.svg"
      />
      <h2>
        Visi贸n
      </h2>
      <p>
        Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod
		tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam,
		quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo
		consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse
		cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non
		proident, sunt in culpa qui officia deserunt mollit anim id est laborum.
      </p>
    </div>
    <div
      className="mission-container"
    >
      <img
        alt="OIENIV Objetivo"
        src="/static/images/eye.svg"
      />
      <h2>
        Objetivo
      </h2>
      <p>
        Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod
		tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam,
		quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo
		consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse
		cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non
		proident, sunt in culpa qui officia deserunt mollit anim id est laborum.
      </p>
    </div>
  </div>
  <style
    jsx={true}
  >
    
					#mission-main {
						padding: 50px 0;
					}
					.mission-container {
						width: 23.3%;
						background: red;
						display: inline-block;
						margin: 0 5%
					}
				
  </style>
</div>
`;
